recipes/oslo.concurrency/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "oslo.concurrency" %}
```

Comment:
Hmm I didn't think "." was allowed in package names. CC @conda-forge/staged-recipes

Reply:
I think "." can be allowed here in package name. Example-
https://github.com/conda-forge/oslo.config-feedstock/blob/master/recipe/meta.yaml
https://github.com/conda-forge/oslo.utils-feedstock/blob/master/recipe/meta.yaml

Reply:
Is there a non-oslo package example?

Reply:
ruamel.yaml and backports packages

Reply:
OK